kafka_utils/kafka_cluster_manager/cmds/command.py

```diff
@@ -200,10 +200,6 @@ def get_reduced_assignment(
             if set(replica) != set(new_assignment[t_p])
         ]
 
-        if (leaders_changes <= max_leader_only_changes and
-                partition_change_count <= max_partition_movements):
-            return new_assignment
```

Comment:
`leaders_changes` and `partition_change_count` are lists, but they're being compared to integers.

In py3, this is no longer valid. When I changed them to compare the lengths of the list to the integers, it no longer worked, so I just removed them. The tests still passed 🤔